v_3_0_0/master_template.go

```diff
@@ -644,14 +644,25 @@ write_files:
                         - nginx-ingress-controller
                   topologyKey: kubernetes.io/hostname
           serviceAccountName: nginx-ingress-controller
+          initContainers:
+          - command:
+            - sh
+            - -c
+            - sysctl -w net.core.somaxconn=32768; sysctl -w net.ipv4.ip_local_port_range="1024 65535"
```

Comment:
@r7vme I remember you did some manual changes for ingress-controller. Were those somehow related to the queue for connections waiting for acceptance?